src/utils/powersystems_utils.jl

```diff
@@ -342,6 +342,29 @@ function is_time_variant(cost_function::PSY.FuelCurve{PSY.PiecewisePointCurve})
     return isa(PSY.get_fuel_cost(cost_function), IS.TimeSeriesKey)
 end
 
+function create_temporary_cost_function_in_system_per_unit(
+    original_cost_function::PSY.CostCurve,
```

Comment:
@rodrigomha do we need to make this a more specific type?

Reply:
It depends. The function just replace the data in the value curve with a new value curve, always assuming that the new point data comes in per-unit, so it does not really matter whatever the original data was from (Incremental or Point or Average).

Alternatively, using something like `PSY.CostCurve{PSY.PiecewisePointCurve}` and `PSY.FuelCurve{PSY.PiecewisePointCurve}` would work for the purposes of this PR, makes to only work if the data comes from a PointCurve.